tests/integration_tests/test_multi_2_sec.py

```diff
@@ -0,0 +1,409 @@
+from openmdao.utils.assert_utils import assert_near_equal
+import unittest
+
+
+class Test(unittest.TestCase):
+    def assert_check_results_sym(self, prob):
```

Comment:
Could you add a quick comment for future reference, something like "we put assert_near_equal here and both test_constraints and test_constructions call this check because these two test cases should give the identical results"

Reply:
Sure